Makefile

```diff
@@ -128,9 +132,8 @@ format: $(GOIMPORTS)
 
 # proto generates golang files from Thanos proto files.
 .PHONY: proto
-proto: deps
-	@go install ./vendor/github.com/gogo/protobuf/protoc-gen-gogofast
-	@./scripts/genproto.sh
+proto: deps $(GOIMPORTS) $(PROTOC)
+	@GOIMPORTS_BIN="$(GOIMPORTS)" PROTOC_BIN="$(PROTOC)" ./scripts/genproto.sh
```

Comment:
Nice :+1:

Reply:
but maybe positional arguments would work better? Not a blocker

Reply:
Sorry, I can't catch your suggestion.